### PR TITLE
RHDEVDOCS-5860: Changing the text of the button

### PR DESCRIPTION
**Pull request by shipsing**
Version(s):

Issue:

Link to docs preview:

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by Dhruv-Soni11** [comment]
> Version(s):
> 1.11
> Issue:
> https://issues.redhat.com/browse/RHDEVDOCS-5860
> Link to docs preview:
> 
> QE review:
> 
> * [ ]  QE has approved this change.
> 
> Additional information:

**Comment by Dhruv-Soni11** [comment]
> Version(s):
> GitOps 1.11
> Issue:
> https://issues.redhat.com/browse/RHDEVDOCS-5860
> Link to docs preview:
> 
> QE review:
> 
> * [ ]  QE has approved this change.
> 
> Additional information:

**Review by nebelsprachen — APPROVED** [review]
lgtm

**Comment by varshab1210** [comment]
LGTM, thanks

**Review by reginaelyse — APPROVED** [review]

File: modules/gitops-argo-cd-installation.adoc [hunk 1]
```diff
@@ -17,9 +17,9 @@ To manage cluster configurations or deploy applications, you can install and dep
 
 . Select *OpenShift GitOps Operator* from the installed operators and select the *Argo CD* tab.
 
-. Click *Create* to configure the parameters:
+. Click *Create ArgoCD* to configure the parameters:
 
-.. Enter the *Name* of the instance. By default, the *Name* is set to *argocd*. 
+.. Enter the *Name* of the instance. By default, the *Name* is set to *example*.
```
Comment (eromanova97) [inline modules/gitops-argo-cd-installation.adoc:22]:
```suggestion
.. Enter the *Name* of the instance. By default, the *Name* is set to `example`. 
```
For reference: https://redhat-documentation.github.io/supplementary-style-guide/#text-entry
Also here: https://www.ibm.com/docs/en/ibm-style?topic=format-highlighting

**Review by eromanova97 — COMMENTED** [review]
Hello, I have just one minor comment, I commented more on this procedure in my review here: https://github.com/openshift/openshift-docs/pull/71605#discussion_r1502672899
Thank you!

File: modules/gitops-argo-cd-installation.adoc (hunk 1 above)
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc:22]: Fixed. PTAL.

Comment (eromanova97) [reply modules/gitops-argo-cd-installation.adoc]: LGTM, thank you!

**Review by nebelsprachen — APPROVED** [review]
lgtm. Just one suggestion based on the IBM style guide

File: modules/gitops-argo-cd-installation.adoc [hunk 2]
```diff
@@ -17,9 +17,9 @@ To manage cluster configurations or deploy applications, you can install and dep
 
 . Select *OpenShift GitOps Operator* from the installed operators and select the *Argo CD* tab.
```
[the remark this answers is not in the document]
Comment (Dhruv-Soni11) [reply modules/gitops-argo-cd-installation.adoc]: Fixed.

**Review by mletalie — COMMENTED** [review]
/label peer-review-in-progress

**Review by mletalie — COMMENTED** [review]
/remove-label peer-review-needed
/remove-label peer-review-in-progress
/label peer-review-done
/lgtm

**Pull request merged**